<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="Ant suite" duration-ms="104" started-at="2016-12-12T16:26:14Z" finished-at="2016-12-12T16:26:14Z">
    <groups>
    </groups>
    <test name="Ant test" duration-ms="104" started-at="2016-12-12T16:26:14Z" finished-at="2016-12-12T16:26:14Z">
      <class name="hello.HelloWorldTest">
        <test-method status="FAIL" signature="testWillAlwaysFail()" name="testWillAlwaysFail" duration-ms="29" started-at="2016-12-12T16:26:14Z" finished-at="2016-12-12T16:26:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[An error message]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: An error message
	at org.testng.Assert.fail(Assert.java:89)
	at hello.HelloWorldTest.testWillAlwaysFail(HelloWorldTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1125)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1007)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:932)
	at org.testng.TestNG.run(TestNG.java:868)
	at org.testng.TestNG.privateMain(TestNG.java:1150)
	at org.testng.TestNG.main(TestNG.java:1114)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="testWillIncreaseCoverage()" name="testWillIncreaseCoverage" duration-ms="0" started-at="2016-12-12T16:26:14Z" finished-at="2016-12-12T16:26:14Z">
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
